app/src/lib/stores/repositories-store.ts

```diff
@@ -132,6 +133,7 @@ export class RepositoriesStore extends TypedBaseStore<
         ? await this.findGitHubRepositoryByID(repo.gitHubRepositoryID)
         : await Promise.resolve(null), // Dexie gets confused if we return null
       repo.missing,
+      enableRepositoryAliases() ? repo.alias : null,
```

Comment:
This prevents the alias from being loaded from persistence, so we can forget about checking the feature flag in many other places 😅